lib/yaml_encoder.ex

```diff
@@ -82,12 +82,22 @@ defmodule YamlEncoder do
   defp encode_key_value indent, {k, v} do
     prefix = indent_spaces indent
 
-    if is_map(v) || is_list(v) do
-      value = encode indent + 1, v
-      "#{prefix}#{k}:\n#{value}"
-    else
-      value = encode indent, v
-      "#{prefix}#{k}: #{value}"
+    case v do
+      v when is_map(v) or is_list(v) ->
+        value = encode indent + 1, v
+        "#{prefix}#{k}:\n#{value}"
+      v when is_binary(v) ->
+        if String.contains?(v, "\n") do
+            value_prefix = indent_spaces indent+1
```

Comment:
can you please indent this line? I think `value_prefix` is indented with 4 spaces instead of 2